README.rst

```diff
@@ -70,9 +70,110 @@ We appreciate any help!
 BUILD & DEVELOPMENT
 -------------------
 
-Run ``python3 setup.py install``
+1. Installation instructions for **Windows** users:
 
-Note: You need to have bower installed before you may install the python packages.
```

Comment:
why do we need bower installed before the python packages?

Reply:
I apologize, it's not mandatory to install bower before python packages. Will remove this